pkg/kubectl/cmd/create_secret.go

```diff
@@ -171,6 +171,8 @@ func NewCmdCreateSecretDockerRegistry(f cmdutil.Factory, cmdOut io.Writer) *cobr
 	cmd.Flags().String("docker-email", "", i18n.T("Email for Docker registry"))
 	cmd.Flags().String("docker-server", "https://index.docker.io/v1/", i18n.T("Server location for Docker registry"))
 	cmd.Flags().Bool("append-hash", false, "Append a hash of the secret to its name.")
+	cmd.Flags().StringSlice("from-file", []string{}, "Key files can be specified using their file path, in which case a default name will be given to them, or optionally with a name and file path, in which case the given name will be used.  Specifying a directory will iterate each named file in the directory that is a valid secret key.")
+	cmd.Flags().StringArray("from-literal", []string{}, "Specify a key and literal value to insert in secret (i.e. mykey=somevalue)")
```

Comment:
`--from-file` does make sense here, `--from-literal` does not. We already provide specific commands for these (`docker-username`, `docker-password`...) so no need from using the generic literal flag. This will result only in wrongly created docker secret.

Reply:
Thanks, that makes sense. Will remove that flag